Code/Legacy/CrySystem/LevelSystem/SpawnableLevelSystem.cpp

```diff
@@ -542,15 +539,53 @@ namespace LegacyLevelSystem
     }
 
 #if defined (CARBONATED)
+    bool SpawnableLevelSystem::ApplyLevelLoadScreenConfig(const char* levelName) const
+    {
+        AZStd::vector<AZStd::string> levelNamesForExclude;
+        ICVar* disabledLevelsVar = gEnv->pConsole->GetCVar("list_of_levels_without_load_screen");
+        const AZStd::string disabledLevels = disabledLevelsVar ? disabledLevelsVar->GetString() : "";
+        if (disabledLevels != "")
+        {
+            AZ::StringFunc::Tokenize(AZStd::string_view(disabledLevels), levelNamesForExclude, ",");
+        }
+
+        const AZStd::string levelNameStr(levelName);
+        for (const auto& nameForExclude : levelNamesForExclude)
+        {
+            if (levelNameStr.contains(nameForExclude))
+            {
+                return false; // level does not have a load screen
+            }
+        }
```

Comment:
We should not need to do this at all.  If you're already checking whether or not the file exists, we can make a simple rule that says:
for every level that is loaded, check to see if levels/level/level.cfg exists.  If it does, load it.  Otherwise, we load default_level_config_file.cfg
(keeping it generic, as that allows us to add additional per-level configuration, beyond loading screen params)

Reply:
I did absolutely the same as you described above.
But we have two special non-battle levels which are loaded at the same code place with other levels:
1) "launch"
2) "worldmap"
I can't hardcoded their names into engine code to disable loading screen for them. And I added "list_of_levels_without_load_screen" cvar.
Ok, I will try to do something to avoid using of "list_of_levels_without_load_screen" cvar.
In the past I tried to enable the loading screen for "worldmap" level and it worked but looked slightly unusual. 
For "launch" level the loading screen crashed on tick processing and progress bar was not calculated, due to many things were being initialized at the same time and they were not ready.

Reply:
I see.  In the LY game we had a loading screen going to the worldmap and launch.  Understand that there's issues with the launch screen, but worldmap should have one too.

instead of this list that's maintained somewhere, we can just use the level loading screen cvars - if the path is overriden to be empty, no loading screen.

so then the launch screen can define an empty path (no loading screen), everything else will use the default, and specific levels can override if they want to use their own.

this eliminates the need to maintain a list

Reply:
Added a simple missed checking to prevent crash during game loading (Added gEnv->pGame for non-null verification). It works on PC. Waiting Jenkins build for iOS verification.

Reply:
@Twolewis : iOS verified via Build #26528 from Jenkins. 
The game does not crash on the launch. Worldmap progress bar appears.